toc/rfc/app-runtime-interfaces/rfc-draft-user-creation-by-org-managers.md

```diff
@@ -0,0 +1,63 @@
+# Meta
+[meta]: #meta
+- Name: User Creation by Org Managers
+- Start Date: 2024-08-13
+- Author(s): stephanme
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: [#946](https://github.com/cloudfoundry/community/pull/946)
+
+
+## Summary
+
+Allow Org Managers to create users in UAA in order to improve the onboarding procedure for larger developer groups into multi-tenant Cloud Foundry foundations.
```

Comment:
Two things it might be nice to include in the summary (assuming they're true):
1. CC is creating the user on behalf of the Org Manager
2. CC admin users can also use this

Reply:
Done